recipes/cspice/all/conanfile.py

```diff
@@ -14,8 +14,16 @@ class CspiceConan(ConanFile):
     exports = ["TSPA.txt"]
     generators = "cmake"
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_utilities": [True, False]
```

Comment:
I would name the option `utilities`.
*build_utilities` is semantically only correct while building :smile: